src/resources/Enums.ts

```diff
@@ -1200,6 +1200,8 @@ export enum ApiKeyStatusFilter {
     SOON_TO_BE_EXPIRED = 'SOON_TO_BE_EXPIRED',
     /** Keys that are active but have been flagged as exposed */
     ACTIVE_AND_EXPOSED = 'ACTIVE_AND_EXPOSED',
+    /** All key that are exposed */
+    ALL_EXPOSED,
```

Comment:
Enums in js are assigned a numeric value, you should define the value explicitly (that's why the CI failed)

```suggestion
    ALL_EXPOSED = 'ALL_EXPOSED',
```